docs/data/material/components/tabs/tabs.md

```diff
@@ -67,13 +67,13 @@ The `centered` prop should be used for larger views.
 
 ### Automatic scroll buttons
 
-By default, left and right scroll buttons are automatically presented on desktop and hidden on mobile. (based on viewport width)
+Using the attributes `variant="scrollable"` and `scrollButtons="auto"`, left and right scroll buttons are automatically displayed on desktop, while on mobile they are hidden based on the viewport width.
```

Comment:
Slight nit here, pulling from Sam's suggestion 😬 

```suggestion
Use the `variant="scrollable"` and `scrollButtons="auto"` props to display left and right scroll buttons on desktop that are hidden on mobile:
```

I think the most accurate term is _prop_ rather than _attributes_, so as not to confuse it with HTML attributes.